### PR TITLE
New user state: limited

### PR DESCRIPTION
**Pull request by gentlecat**
Just like `active`, but can't get access to tokens. :no_good:

### CODE REVIEW

File: metabrainz/admin/views.py [hunk 1]
```diff
@@ -53,8 +53,12 @@ def details_stats(self, user_id):
     @expose('/approve')
     def approve(self):
         user_id = request.args.get('user_id')
-        User.get(id=user_id).set_state(STATE_ACTIVE)
-        flash.info("User #%s has been approved." % user_id)
+        if request.args.get('limited'):
+            User.get(id=user_id).set_state(STATE_LIMITED)
+            flash.warn("User #%s has been approved (with limited access)." % user_id)
```
Comment (mayhem) [inline metabrainz/admin/views.py]: Please use the same message for both STATE_LIMITED and STATE_ACTIVE. ("User #%s has been approved.")

Comment (gentlecat) [inline metabrainz/admin/views.py]: Sure, if that's what you want.

Comment (gentlecat) [inline metabrainz/admin/views.py]: Done. :tractor:

File: metabrainz/templates/users/profile.html [hunk 2]
```diff
@@ -94,6 +56,49 @@ <h1 class="page-title">Your Profile</h1>
     </div>
   </div>
 
+  {% if current_user.is_commercial %}
+    {% if current_user.state == 'active' and current_user.good_standing %}
```
Comment (mayhem) [inline metabrainz/templates/users/profile.html]: Use the same block for active and limited. Both types of users should see the same message. Limited users should not see the "generate access token" field below.

File: metabrainz/templates/users/profile.html [hunk 3]
```diff
@@ -94,6 +56,49 @@ <h1 class="page-title">Your Profile</h1>
...
+        You have permission to use any of the data published by the MetaBrainz
+        Foundation. This includes data dumps released under a Creative Commons
+        non-commercial license. Thank you for your support!
+      </p>
+      <p>
+        Please note that if your support falls behind by more than 60 days, this
+        permission may be withdrawn. You can always check your current permission
+        status on this page.
+      </p>
+    {% elif current_user.state == 'limited' %}
```
Comment (mayhem) [inline metabrainz/templates/users/profile.html]: Nuke this block.

Comment (gentlecat) [inline metabrainz/templates/users/profile.html]: Done. :ok_hand: :boom:

**Comment by mayhem** [comment]
:fish_cake: :secret: :cn: :boom:

**Comment by gentlecat** [comment]
Please don't send me to China again. :es: :walking:

**Comment by mayhem** [comment]
LOL, you can stay. :) :cn: :gun:

**Pull request merged**